src/orquestra/quantum/evolution.py

```diff
@@ -62,19 +61,17 @@ def time_evolution_for_term(
 
     if len(term.terms) != 1:
```

Comment:
nitpick (non-blocking): do we need this check anymore?

Reply:
Fixed in [f9c6d97](https://github.com/zapatacomputing/orquestra-quantum/pull/47/commits/f9c6d9757b7201a2f3bca0c7ded7d550beadf146)